### PR TITLE
Mf/iOsObjectMapperSetup

### PR DESCRIPTION
**Pull request by mwflickner**

### CODE REVIEW

File: clients/iOS/Grapevine/Grapevine/Event.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    var title: String!
+    //var date: NSDate
+    //var status: String
+    //var id: Int
+    //var timeProcessed: NSDate
+    //var location: Location
+    //var startTime: NSDate
+    //var endTime: NSDate
+    //var repeatsWeekly: Bool
+    //var tags: [String]
```
Comment (rachelriv) [inline clients/iOS/Grapevine/Grapevine/Event.swift:22]: Why are all of these commented out?

**Comment by rachelriv** [comment]
Looks like you put a lot of work into this, which is awesome! :+1: 

Would you mind providing a short explanation about what this PR does? Does it just map the JSON from responses into an object?

File: clients/iOS/Grapevine/Grapevine/Event.swift (hunk 1 above)
Comment (mwflickner) [inline clients/iOS/Grapevine/Grapevine/Event.swift:22]: Just simplifying when I was setting up ObjectMapper. They'll be good to go in my next PR

**Comment by mwflickner** [comment]
essentially yes it will map JSON to objects and objects to JSON

**Pull request merged**